climada/entity/impact_funcs/base.py

```diff
@@ -127,4 +127,65 @@ def check(self):
                            'origin of the intensity scale. In impact.calc '
                            'the impact is always null at intensity = 0.')
 
+    def set_step_ImpF(self, threshold, inten_min, inten_max, if_id=1):
+
+        """ Step function type impact function. Everything is destroyed above
+        threshold. Usefull for high resolution modelling.
+        
+        This method modifies self (climada.entity.impact_funcs instance)
+        by assining an id, intensity, mdd and paa to the impact function.
+        
+        Parameters
+        ----------
+            threshold : float
+                threshold over which exposure is fully destroyed
+            inten_min : float
+                minimum value of intensity range
+            inten_min : float
+                maximum value of intensity range
+            if_id : int, optional, default=1
+                impact function id
+
+        """
 
+        self.id = if_id
+        self.intensity = np.array([inten_min, threshold, threshold, inten_max])
+        self.paa = np.array([1, 1, 1, 1])
+        self.mdd = np.array([0, 0, 1, 1])
+
+    def set_sigmoid_ImpF(self, sig_mid, sig_shape, sig_max,
+                    inten_min, inten_max, inten_step=5, if_id=1):
```

Comment:
if the first function is modified that way, I'd also replace the `inten_min`, `inten_max`, `inten_step` with 
`intensity: tuple(float, float, float)
	            tuple of 3-intensity numbers: (minimum, threshold, maximum)`

Reply:
I'm adjusting to the tuple, but I do (minimum, maximum, step) to be in line with the np.arange naming of things

Reply:
see 944f9aa7